js/config.js

```diff
@@ -51,13 +52,15 @@ function readUserConfig() {
 
         fs.readFile(configPath, 'utf8', function(err, data) {
             if (err) {
+                log.send(logLevels.ERROR, 'config: cannot open user config file: ' + configPath + ', error: ' + err);
```

Comment:
i don't think we need all of these log message where we already doing a promise reject.  this should be handled at a higher level.

ditto for all the other below here in this file.

what do others think?